pytest.ini

```diff
@@ -3,26 +3,3 @@ addopts = -v -p no:warnings
 testpaths = tests
 markers = 
     serial: Tests that must be ran without the -n argument
-
-
-# TODO - From where does the "env" option come?  See #1104.
```

Comment:
this deletion will clear ticket 1104, which I'll make a not to close when this PR is merged.